config/locales/it.yml

```diff
@@ -0,0 +1,741 @@
+    invitations_controller:
+      flashpublicinvitnotallowed: "Richieste di inviti pubbliche non sono permesse."
+      flashsuccessinvitrequest: "La tua richiesta d'invito è stata validata e sarà ora visibile agli altri utenti connessi."
+      flashaccountnotinvit: "Il tuo profilo non è autorizzatto a condividere inviti"
+      flashsuccessinvit: "L'email d'invito è stata inviata con successo a %{guest}."
+      flasherrorssinvit: "Non è stato possibile inviare l'invito, si verifichi la validità dell'indirizzo e-mail."
+      flashrequestconfirmation: "Ti è stata inviata una conferma a %{email}."
+      flashinvalidinvitation: "Invito invalido o scaduto"
```

Comment:
in carrozzina.

Tradurrei come "non valido"

Reply:
In carrozzina?
Va bene il "non valido".

Reply:
gli invalidi in Italiano sono quelli con menomazioni fisiche. Non è come invalid in inglese.

Reply:
Stando a [Treccani](http://www.treccani.it/vocabolario/invalido/) (sez.2) è valido il significato adoperato.

Reply:
Ci sta comunque meglio non valido

Reply:
è molto desueto, il significato colloquiale è ben altro. Andiamo con "non valido".

Reply:
Non sono d'accordo per quanto riguarda la desuetudine del termine; d'accordo, comunque.